docs/best-practice/limit-and-offset-vs-cursor-api-and-hasmore.md

```diff
@@ -0,0 +1,166 @@
+---
+sidebar_position: 50
+title: LIMIT and OFFSET vs Cursor API and hasMore
+---
+
+Queries that return large volumes of data may require more than one response to provide the complete results. A common method is using the `LIMIT` and `OFFSET` statements in the query. However, this is not the optimized approach. When these options are used, each time the query is invoked query processing is performed. 
```

Comment:
This introduction does not give me any idea what to expect as I read on through the text. If feel like I got dropped in the deep end of a pool blindfolded. Can you use simpler language to start out and explain what is going on?

I gather that you are talking to someone using queries that return large volumes of data, perhaps something like "Manage Large Query Returns" as a title?